src/components/Sections/SectionChart/ChartLegend.js

```diff
@@ -30,24 +30,20 @@ const ChartLegend = ({ data, icon = 'circle', layout = CHART_LAYOUT_TYPE.vertica
     const mainClass = `recharts-legend-item legend-item-${i} ${layout}`;
     const legendIconClass = `${icon} icon chart-legend-icon`;
     let width = 'auto';
-    let maxWidth;
     const value = group.value ? numberToShortString(group.value) : null;
     // decrease width of name (if value exists) to allow for ellipsis.
     if (value && showValue) {
       let valueInPixels = (value + '').replace('.', '').length * DIGIT_PIXEL_SIZE + ICON_CONTAINER_PIXEL_SIZE;
       if (valueDisplay === VALUE_FORMAT_TYPES.stretch) {
         width = `calc(100% - ${valueInPixels}px)`;
-      } else if (valueDisplay === VALUE_FORMAT_TYPES.minimal) {
```

Comment:
we don't support it anymore?

Reply:
not needed for reports. just causing issues, ellipsis not needed